sdk/textanalytics/Azure.AI.TextAnalytics/src/ServiceClients/LanguageServiceClient.cs

```diff
@@ -3,8 +3,12 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
```

Comment:
This includes all work that was merged into the feature branch at the time the PR was opened.   I'll do another rebase before we take next steps to pull in any changes after that point.